sdmetrics/reports/single_table/_properties/column_pair_trends.py

```diff
@@ -123,7 +123,7 @@ def _get_processed_data(self, data, metadata):
                     column_name, processed_data[column_name]
                 )
 
-        return processed_data, pd.DataFrame(discretized_dict)
+        return processed_data, pd.DataFrame(discretized_dict, index=processed_data.index)
```

Comment:
Just checking, is there always an `.index` provided?

Reply:
The method expects a pandas dataframe as input, so I guess there will always be a `.index` if this is the question